streams/jp.m3u

```diff
@@ -1,118 +1,42 @@
-#EXTINF:-1 tvg-id="CartoonNetwork.jp",Cartoon Network Japan (400p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs25
+#EXTINF:-1 tvg-id="ATX.jp",AT-X (720p)
+http://asia.vthanhtivi.pw/AT-X/index.m3u8
+#EXTINF:-1 tvg-id="BS11.jp",BS11 (720p)
+http://asia.vthanhtivi.pw/BS11/index.m3u8
 #EXTINF:-1 tvg-id="CGNTVJapan.jp",CGNTV Japan (1080p)
 https://cgntvliveweb-944.acs.wecandeo.com/ms/2850/944/index.m3u8
 #EXTINF:-1 tvg-id="CGNTVJapan.jp",CGNTV Japan (720p)
 http://cgntv-glive.ofsdelivery.net/live/cgntv_jp/playlist.m3u8
-#EXTINF:-1 tvg-id="ChannelGinga.jp",Channel Ginga (1080p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs29
-#EXTINF:-1 tvg-id="CNNj.jp",CNNj (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs16
-#EXTINF:-1 tvg-id="DisneyChannel.jp",Disney Channel Japan (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=bs24
-#EXTINF:-1 tvg-id="DisneyJunior.jp",Disney Junior (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs23
-#EXTINF:-1 tvg-id="EiseiGekijo.jp",Eisei Gekijo (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs22
-#EXTINF:-1 tvg-id="FamilyGekijyo.jp",Family Gekijo (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs20
-#EXTINF:-1 tvg-id="FishingVision.jp",Fishing Vision (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=bs25
 #EXTINF:-1 tvg-id="JOCXDTV.jp",Fuji TV (720p)
 http://203.162.235.41:15094
 #EXTINF:-1 tvg-id="JOCXDTV.jp",Fuji TV (720p)
 http://asia.vthanhtivi.pw/Fuji_TV/index.m3u8
-#EXTINF:-1 tvg-id="JOCXDTV.jp",Fuji TV (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=gd05
 #EXTINF:-1 tvg-id="JOCXDTV.jp",Fuji TV (540p)
 https://fujitv4.mov3.co/hls/fujitv.m3u8
-#EXTINF:-1 tvg-id="FujiTVNEXT.jp",Fuji TV NEXT (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs26
 #EXTINF:-1 tvg-id="",Gaki no Tsukai (English Subs) (720p)
 https://hamada.gaki-no-tsukai.stream/hls/test.m3u8
-#EXTINF:-1 tvg-id="GAORASPORTS.jp",Gaora Sports (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs17
 #EXTINF:-1 tvg-id="GolfNetwork.jp",Golf Network (720p)
 http://203.162.235.41:28107
-#EXTINF:-1 tvg-id="GolfNetwork.jp",Golf Network (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=cs03
 #EXTINF:-1 tvg-id="GolfNetwork.jp",Golf Network (540p)
 http://202.60.106.14:8080/1335/playlist.m3u8
-#EXTINF:-1 tvg-id="GreenChannel.jp",Green Channel (544p)
-http://cdns.jp-primehome.com:8000/zhongying/live/playlist.m3u8?cid=bs14
 #EXTINF:-1 tvg-id="GSTV.jp",GSTV (720p)
 https://gstv-tnz-gsmediastreaming.preview-jpea.channel.media.azure.net/dfd06b62-e9d1-4a7f-bcbb-89d2ecbc82ee/preview.ism/manifest(format=mpd-time-csf,audio-only=false)
 #EXTINF:-1 tvg-id="JOMLDTV.jp",Gunma TV (720p)
 https://movie.mcas.jp/switcher/smil:mcas8.smil/master.m3u8
```

Comment:
dead

Reply:
Appears to be [working](https://streamtest.in/logs/http-cdns-jp-primehome-com-8000-zhongying-live-playlist-m3u8-cid-bs14),

Reply:
it's gunma TV that has moved